apps/wa/wa-messages-set-processed-state-messages/aat/01.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: wa-messages-set-processed-state-messages
+spec:
+  values:
+    job:
+      schedule: "30 * * * *"
+      environment:
+        JOB_NAME: SET_STATE_TO_PROCESSED_ON_MESSAGES
+        RE_RUN: true
```

Comment:
what does `RE_RUN` do?  It's false in cluster 00 and true here.  Is that correct?

Reply:
I don't know off the top of my head.  Please take a look at the code to see what this env var is used for so we know why we are adding it.  We ought to avoid copy / paste unless we know what it is doing.

Reply:
As discussion with Rasika and Ravi, this dummy env variable (RE_RUN) is used to forcibly restart the pod, it means that the new message job is necessary to keep it, we will remove this variable from aat/00.yaml and aat/01.yaml